internal/test/wmcb/main_test.go

```diff
@@ -17,7 +18,14 @@ var (
 )
 
 func TestMain(m *testing.M) {
-	err := framework.Setup(vmCount)
+	var vmCreds e2ef.Creds
+	var skipVMSetup bool
+
+	flag.Var(&vmCreds, "vmCreds", "List of VM credentials")
+	flag.BoolVar(&skipVMSetup, "skipVMSetup", false, "Option to disable setup in the VMs")
+	flag.Parse()
+
+	err := framework.Setup(vmCount, vmCreds, skipVMSetup)
```

Comment:
nit: Why to pass the `skipVMSetup` arg here? Why can't we set up the framework to use the variable directly here? https://github.com/openshift/windows-machine-config-operator/pull/122/files#diff-f742eafb78aace92926680fb9193bfe9R14? Ideally I'd like to see minimal args to setup, none if it's ideal.

Reply:
Why should I add `skipVMSetup` to the `framework` struct when I am only using it in `Setup()`?